snippets/language-asciidoc.cson

```diff
@@ -96,160 +96,175 @@
     prefix: '=4'
     body: '''
       ==== $1
 
       $0
       '''
-  'Heading 4':
-    description: 'Heading 4 (one-liner)'
+  'Section 4':
+    description: 'Level 4 section (single-line syntax)'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#sections'
     prefix: '=5'
     body: '''
       ===== $1
 
       $0
       '''
+  'Section 5':
+    description: 'Level 5 section (single-line syntax)'
+    descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#sections'
+    prefix: '=6'
+    body: '''
+      ====== $1
+
+      $0
+      '''
 
-  # Macro
+  # Inline Macros
   #
   'Keyboard':
-    description: 'Keyboard shortcuts'
+    description: 'Keyboard shortcut'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#keyboard-shortcuts'
-    prefix: '/kbd'
+    prefix: 'kbd'
     body: 'kbd:[$1] $0'
   'Menu selection':
-    description: 'Menu Selection'
+    description: 'Menu selection'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#menu-selections'
-    prefix: '/mn'
+    prefix: 'menu'
     body: 'menu:$1[$2] $0'
   'UI buttons':
-    description: 'UI buttons'
+    description: 'UI button'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#ui-buttons'
-    prefix: '/btn'
+    prefix: 'btn'
     body: 'btn:[$1] $0'
   'Inline Image':
-    description: 'Inline Image'
+    description: 'Inline image'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#images'
-    prefix: '/img'
-    body: 'image:$1[$2] $0'
+    prefix: 'img'
+    body: 'image:${1:target}[${2:alt}] $0'
   'Link macro':
-    description: 'Link macro'
+    description: 'Explicit link'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#url'
-    prefix: '/lk'
-    body: 'link:${1:url}[$2] $0'
+    prefix: 'link'
+    body: 'link:${1:url}[${2:label}] $0'
   'mailto link':
-    description: 'mailto link'
+    description: 'E-mail link'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#url'
-    prefix: '/mt'
-    body: 'mailto:${1:email}[$2] $0'
+    prefix: 'mail'
+    body: 'mailto:${1:email}[${2:label}] $0'
   'stem':
-    description: 'stem'
+    description: 'STEM expression'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#activating-stem-support'
-    prefix: '/stem'
-    body: 'stem:$2[$1] $0'
+    prefix: 'stem'
+    body: 'stem:[${1:expr}] $0'
   'stem asciimath':
-    description: 'stem asciimath'
+    description: 'AsciiMath STEM expression'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#activating-stem-support'
-    prefix: '/asciimath'
-    body: 'asciimath:$2[$1] $0'
+    prefix: 'asciimath'
+    body: 'asciimath:[${1:expr}] $0'
   'stem latexmath':
-    description: 'stem latexmath'
+    description: 'LaTeX STEM expression'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#activating-stem-support'
-    prefix: '/latexmath'
-    body: 'latexmath:$2[$1] $0'
+    prefix: 'latex'
```

Comment:
Technically, there is no such thing as LaTeX Math. It's just LaTeX. AsciiMath is an official name. So we're choosing the shortest word that will apply. (Arguably, AsciiDoc should use latex instead of latexmath, but c'est la vie).